package.json

```diff
@@ -238,10 +238,11 @@
 		"native": "npm run --prefix packages/react-native-editor",
 		"pot-to-php": "./bin/pot-to-php.js",
 		"postinstall": "patch-package && node ./patches/patch-xcode.js",
+		"prepublishOnly": "npm run clean:package-types && npm run build:packages",
 		"publish:check": "lerna updated",
-		"publish:dev": "npm run clean:package-types && npm run build:packages && lerna publish prerelease --preid rc --dist-tag next",
-		"publish:patch": "npm run clean:package-types && npm run build:packages && lerna publish --dist-tag patch",
-		"publish:prod": "npm run clean:package-types && npm run build:packages && lerna publish",
+		"publish:dev": "lerna publish prerelease --preid rc --dist-tag next",
```

Comment:
I think we will be able to remove `publish:dev` with the `isPrerelease` option that is now fully supported.